src/plugins/QuicTrace.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29102.190
+# Visual Studio Version 17
+VisualStudioVersion = 17.1.31911.260
```

Comment:
I believe this automatically happened when I opened in VS 2022 (Preview). Should we check this in?

Reply:
I think its fine. I imagine most users will be moving to 2022. Its difficult to get 2019 anymore (They hide the downloads).